source/api/image/3.0/change-log.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Image API 3.0 Change Log"
+title_override: "Changes for IIIF Image API Version 3.0"
+id: image-api-30-change-log
+layout: spec
+cssversion: 3
+tags: [specifications, image-api, change-log]
+major: 3
+minor: 0
+patch: 0
+pre: beta
+redirect_from:
+  - /api/image/3.0/change-log-30.html
+---
+
+This document is a companion to the [IIIF Image API Specification, Version 3.0][image30]. It describes the changes to the API specification made in this major release, including ones that are backwards incompatible with version 2.1.1, the previous version.
+
+## Table of Contents
+{:.no_toc}
+
+* Goes Here
+{:toc}
+
+
+## 1. Breaking Changes
+
+The following changes are backwards incompatible with version 2.1.1 of the Image API.
+
+### 1.1. Image Request Changes
+
+#### 1.1.1. Size `full` has been removed in favor of `max`
+
+Per the [deprecation warning][image21-full-dep] in the previous version, the value `full` is no longer allowed for the size parameter, `max` must be used instead. See issue [#678](https://github.com/IIIF/api/issues/678).
+
+#### 1.1.2. Canonical form of size parameter changed to _`w,h`_
+
+Per the [deprecation warning][image21-full-dep] and the [inconsistency warning][image21-size-inconsistency] about the size parameter in the previous version, the canonical form of the size parameter is now _`w,h`_ unless `max` is requested. This resolves the inconsistency between the server-preferred values in the `sizes` object, and the canonical form of the size parameter. In order to request preferred sizes, a client simply uses `width` and `height` values from `sizes` to build the _`w,h`_ size to request. See issues [#544](https://github.com/IIIF/api/issues/544) and [#678](https://github.com/IIIF/api/issues/678).
+
+#### 1.1.3. Size parameter value must not scale image to larger than the extracted region
+
+The value of the size parameter must not result in an image larger than the extracted region. Attempts to do so must generate an error response. Previous versions allowed implementations to [optionally support scaling up][image21-size]. The `sizeAboveFull` feature name was also removed. See issue [#693](https://github.com/IIIF/api/issues/693).
+
+### 1.2. Image Information Changes
+
+Several existing properties were renamed for consistency with the Presentation API, developer convenience, or to better reflect the intended semantics. Some of the semantics were also clarified based on implementation experience during previous versions.
+
+#### 1.2.1. Rename `@id` to `id`, `@type` to `type`
+
+These properties were renamed to enable javascript developers to use the "dot notation" (`manifest.id`) instead of the square brackets based equivalent needed with the `@` character (`manifest['@id']`). This follows JSON-LD community best practices established by schema.org, the JSON-LD, Web Annotation and Social Web working groups. See issue [#590](https://github.com/IIIF/api/issues/590).
```

Comment:
Perhaps change `manifest` to `image` in the example given the current context

Reply:
Capitalize 'Javascript'

Reply:
:+1: x 2

Reply:
And do something (?) with hyphens in 'square brackets based' (as in Prezi)